### PR TITLE
Feat/display entity list

### PR DESCRIPTION
**Pull request by Joanna-YJA**
This pull request changes the User Interface to make sure that the Entity List of correct Entity type is shown as the result of the Commands. This build on the previous pull request, do ignore some of the files changed.

## Checks
- [] No errors when running the tests
- [] Build and checkstyle passes
- [] Run the app and ran 2 commands without failing
- [] Written the baseline test cases for the PR

## Changelog
- []
- []
- []

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 1]
```diff
@@ -4,26 +4,48 @@
...
 /**
  * Represents the result of a command execution.
  */
 public class CommandResult {
 
     private final String feedbackToUser;
 
-    /** Help information should be shown to the user. */
+    /**
+     * Help information should be shown to the user.
```
Comment (brianyenna) [inline src/main/java/seedu/address/logic/commands/CommandResult.java]: Hey. I'm not sure whether there is a style/standard for this, but I was under the impression that for individual attributes, it's usually the // single line comment. Was there some kind of rule set for this? If there is, could you refer me to it, then i can edit the comments and javadocs for my section of the code :)

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -160,15 +168,16 @@ private void handleExit() {
...
+    public EntityListPanel getListPanel() {
+        return listPanel;
     }
 
     /**
      * Executes the command and returns the result.
      *
      * @see seedu.address.logic.Logic#execute(String)
      */
+    @SuppressWarnings("checkstyle:Regexp")
```
Comment (brianyenna) [inline src/main/java/seedu/address/ui/MainWindow.java]: Hey. Can I just check why this annotation is necessary?

File: src/main/java/seedu/address/logic/commands/CommandResult.java (hunk 1 above)
Comment (Joanna-YJA) [reply src/main/java/seedu/address/logic/commands/CommandResult.java]: I'll ask during tomorrow's tutorial, but this is given under the orignal AddressBook 3 code haha

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (Joanna-YJA) [reply src/main/java/seedu/address/ui/MainWindow.java]: I think I might have accidentally suppressed checkstyle for that to make my build pass.

**Pull request merged**